test/browser_tests/cucumber-new/features/suites/apps/credit_card_agreement_database.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Credit card agreement database
+  As a consumer
+  I want to review general terms and conditions, pricing, and fee information
+  So that I can learn more about credit card agreements.
+
+@ignore_chrome_Linux
+Scenario: Reach the page listing credit card agreements from Bank of America in the database
+  Given I visit the www.consumerfinance.gov/credit-cards/agreements/ URL
+  When I select what I call the "card issuer" drop list and search "Bank of America" and choose first
+  Then I should be directed to the "www.consumerfinance.gov/credit-cards/agreements/issuer/176/" URL
```

Comment:
The URL structure has changed to use a slug based on company name, rather than numeric ID.
```suggestion
  Then I should be directed to the "www.consumerfinance.gov/credit-cards/agreements/issuer/bank-of-america/" URL
```

Reply:
Changed numeric ID to slug based on company name.